core/java/iesi-core/src/test/java/io/metadew/iesi/openapi/ComponentParserTest.java

```diff
@@ -133,7 +133,7 @@ void createComponentWithPathParameter() {
     @Test
```

Comment:
Add test cases for an openapi doc with simple long versions (e.g. 2) and for one with semantic versioning (e.g. 2.3.1) and verify the correct version is being used.